packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -146,13 +146,17 @@ const JurisdictionCard = ({ isAdded = false, isSelected = false, jurisdiction, o
                                         <div className='flex gap-300'>
                                             {!(marketType in verificationDocs)
                                                 ? verificationDocumentsMapper.notApplicable.icon
-                                                : verificationDocs[marketType]?.map(doc => (
-                                                      <JurisdictionCardVerificationTag
-                                                          category={verificationDocumentsMapper[doc].category}
-                                                          icon={verificationDocumentsMapper[doc].icon}
-                                                          key={`verification-doc-${doc}`}
-                                                      />
-                                                  ))}
+                                                : verificationDocs[marketType]?.map(doc => {
```

Comment:
```suggestion
                                                : (doc in verificationDocumentsMapper) && verificationDocs[marketType]?.map(doc => {
```
best to check first before mapping. map will always return an array in each iteration, this might return an array of undefine - [undefined, undefined]

Reply:
The doc in verificationDocumentsMapper check is outside the map function

Reply:
the check is in the map callback 👀 it will become like this if the check doesnt pass
<img width="545" alt="Screenshot 2024-01-19 at 10 42 21 AM" src="https://github.com/binary-com/deriv-app/assets/75345074/d64fbb0d-ef5d-47fe-afff-262dda731476">